### ISSUE TITLE
Bottom inset on Android 15 is 0

### ISSUE THREAD

**Issue by vytautasvargonas** [AppAndFlow/react-native-safe-area-context#546]
On Android 15 (api level 35) the bottom inset returned is 0.
Tested with the gesture navigation and the 3-button navigation

**Comment by foladipo-aforza** [comment]
Yep I am experiencing this too, but only in an Android phone simulator. It works correctly in an Android tablet simulator and in my ios simulators.

**Comment by shadowninja55** [comment]
i am experiencing this on android 14 with a physical device (pixel 7)

**Comment by curtis-jotson** [comment]
Also experiencing this on both physical and emulated Pixel 7. Not experiencing it with Android 14/SDK 34 on any device or iOS.

"expo": "~51.0.38"
"react-native-safe-area-context": "4.10.5"

**Comment by rmargolinross** [comment]
This seems to be a duplicate of https://github.com/th3rdwave/react-native-safe-area-context/issues/552 If you enable Android SDK 35 or edge-to-edge this library returns 0 always for the insets.

**Comment by michelle54725** [comment]
Upgrading to 5.0.0 fixed this for me! I was experiencing all insets being 0 on my physical device (Pixel 8) running Android 15, but not on emulators running either 14/15. After upgrading, top and bottom insets are non-zero 👍

**Comment by michelle54725** [comment]
> Upgrading to 5.0.0 fixed this for me! I was experiencing all insets being 0 on my physical device (Pixel 8) running Android 15, but not on emulators running either 14/15. After upgrading, top and bottom insets are non-zero 👍

Actually all insets are 0 on first open of the app after a fresh install, then it works as expected 😅

**Comment by gavrichards** [comment]
I'm still experiencing this, even on the latest version of this library. Has anyone managed to figure out a workaround?

**Comment by aarontuftss** [comment]
+1

**Comment by jacquesjg** [comment]
Is `android:windowDrawsSystemBarBackgrounds` is set to true?

**Comment by janicduplessis** [comment]
Let's close this one as duplicate of #552

**Comment by gavrichards** [comment]
It's frustrating to see this issue closed as a duplicate, but no acknowledgement of the other issue by the repository's maintainers. Is there anything that can be done here or does anyone have any reliable workarounds?
